docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -4401,6 +4401,42 @@ Content-Type: text/plain; charset=utf-8
 -   **404** - plugin not installed
 -   **500** - plugin is active
 
+### Create a plugin
+
+`POST /plugins/(plugin name)/create`
```

Comment:
@thaJeztah : I went ahead and updated endpoint to `plugin/create/name=<>`, after being convinced about the non-existent object endpoint.